README.md

```diff
@@ -1,2 +1,7 @@
 # divvy
-Hack Reactor Greenfield Project - Expense Splitting App
+#### Hack Reactor Greenfield Project - Expense Splitting App
+
+## Description
+Divvying up expenses between friends and peers can be a disaster. Remember that time you went on a trip and one person rented the car and another paid for dinners and a third bought the gas and the beer and then you had to even-up while accounting for tips? Or that time rent was due but someone had purchased all of the groceries and utilities for the month? Enter Divvy.
```

Comment:
I'm happy to add commas, I was trying to convey the unpleasant nature of divvying expenses up by using a run-on